src/redux/Books/BooksSclice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: {
+    list: [],
+  },
+  reducers: {
+    addBook: (state, action) => {
+      state.list.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.list = state.list.filter((book) => book.id !== action.payload.id);
+    },
+  },
+});
```

Comment:
- [ ] it would be better to export `addBook` and `removeBook`  as actions in order to complete the requirements 
example : `export const { addBook, removeBook } = booksSlice.actions`